src/pages/modules/ModulesPage.slice.ts

```diff
@@ -0,0 +1,138 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import {
+  IModuleState,
+  IModulesGETFailure,
+  IModulesPUT,
+  IModulesPUTFailure,
+  IModule,
+  AlertType,
+} from "../../shared/types";
+import { AlertMsg } from "./config";
+
+const initialModuleState: IModuleState = {
+  modules: [],
+  isLoading: true,
```

Comment:
isLoading should be false by default unless it calls something upon loading - but then that thing it calls should have the responsibility of setting isLoading to true.

Reply:
Got it, changing it